services/object/schemas/v1/download-object-request.yml

```diff
@@ -0,0 +1,17 @@
+$schema: "/schemas/common/metaschema.json#"
+title:              "Download Object Request"
+description: |
+  Download object request. See [Transport Protocols](https://docs.taskcluster.net/docs/reference/platform/object/transport-protocols) for more detail.
+type:               object
+properties:
+  acceptProtocols:
+    type: array
+    title: "Supported protocols"
+    uniqueItems: true
```

Comment:
while it's true this should be unique, do we want to reject the request if it isn't? Maybe better to accept whatever and uniq them in the service? I'm happy either way.

Reply:
We discussed and prefer to enforce uniqueness via the schema rather than add the complexity in the application code and possibly help implementers catch bugs earlier.